src/components/wells/Sample.vue

```diff
@@ -64,12 +68,37 @@ export default {
       this.active = !this.active
       this.replicate.outliers()
     },
+    onMouseOver() {
+      if (this.warningMessage) {
+        // This event bubbles up through Row to Plate, to show the warning message on the page.
+        this.$emit('showWarningMessage', this.warningMessage)
+      }
+    },
+    onMouseLeave() {
+      if (this.warningMessage) {
+        // This event bubbles up through Row to Plate, to hide any warning messages on the page.
+        this.$emit('hideWarningMessage')
+      }
+    },
   },
   mounted() {
     // prevents errors if store is not defined. Is there a better way ...
     if (this.store !== undefined) {
       this.store.qcAssayList.addReplicate(this)
       this.replicate.outliers()
+
```

Comment:
This is definitely leading me to thinking about adding better state management probably Pinia but we don't need to consider it at the moment.